deployment/CP02PMUI_Deploy.csv

```diff
@@ -89,3 +89,12 @@ AR-28,,,,CP02PMUI-WFP01-02-DOFSTK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-0001
 AR-28,,,,CP02PMUI-WFP01-03-CTDPFK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-00011,CGVEH-CWFPCP-93601,CGINS-CTDPFK-00106,40.36498,-70.780298,,70,94,
 AR-28,,,,CP02PMUI-WFP01-04-FLORTK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-00011,CGVEH-CWFPCP-93601,CGINS-FLORTK-00969,40.36498,-70.780298,,70,94,
 AR-28,,,,CP02PMUI-WFP01-05-PARADK000,11,1,2018-04-05T18:24:00,,CGMCP-02PMUI-00011,CGVEH-CWFPCP-93601,CGINS-PARADK-20423,40.36498,-70.780298,,70,94,
+#AR-31,,,,CP02PMUI-RII01-02-ADCPTG010,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,,CGINS-ADCPTG-20499,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-SBS01-00-STCENG000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,,CGCON-BSTC01-00042,40.36,-70.76,,0,94,update post-deployment
+#AR-31,,,,CP02PMUI-SBS01-01-MOPAK0000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,,CGINS-MOPAK0-11363,40.36,-70.76,,0,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-00-WFPENG000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGCON-CWFPCP-71401,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-01-VEL3DK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-VEL3DK-00012,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-02-DOFSTK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-DOFSTK-00209,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-03-CTDPFK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-CTDPFK-00093,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-04-FLORTK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-FLORTK-00830,40.36,-70.76,,70,94,update post-deployment
+#AR-31,,,,CP02PMUI-WFP01-05-PARADK000,12,1,2018-10-20T00:00:00,,CGMCP-02PMUI-00012,CGVEH-CWFPCP-71401,CGINS-PARADK-20525,40.36,-70.76,,70,94,update post-deployment
```

Comment:
Rest of S/Ns reviewed by J. Batryn

Reply:
Confirmed WFP sensor S/Ns

Reply:
Updated MOPAK S/N to 43252 (from 11363) to match config sheet